packages/manager/apps/hub/src/dashboard/products-list/dedicated-cloud.routing.js

```diff
@@ -1,27 +1,22 @@
-import { compact, get, map, merge, pick } from 'lodash-es';
+import { compact, get, map, merge, omit } from 'lodash-es';
 
 import { ListLayoutHelper } from '@ovh-ux/manager-ng-layout-helpers';
-import { urlQueryParams, params, component, resolves } from './config';
+import { component, resolves } from './config';
 
 const routingData = {
-  params,
+  params: ListLayoutHelper.stateParams,
   component,
   resolve: {
     ...resolves,
-    ...pick(ListLayoutHelper.stateResolves, [
-      'onListParamsChange',
-      'filter',
-      'sort',
-      'sortOrder',
-    ]),
-    apiPath: /* @ngInject */ (resourcePath) =>
-      resourcePath.replace(/\/\{[a-zA-Z]+\}/g, ''),
-    schema: /* @ngInject */ ($http, apiPath) =>
-      $http.get(`${apiPath}.json`).then(({ data }) => data),
-    rows: /* @ngInject */ ($http, apiPath, products, propertyId) =>
+    ...omit(ListLayoutHelper.stateResolves, []),
```

Comment:
Why not directly ?
```suggestion
    ...ListLayoutHelper.stateResolves,
```

Reply:
I made some tests and forgot to remove it